js/ParticipantTopDonor.js

```diff
@@ -0,0 +1,43 @@
+$(document).ready(function () {
+    // UpdateInfo immediately, and then once every 10 seconds.
+    UpdateInfo();
+    window.setInterval(UpdateInfo, 10000);
+});
+
+function GetTopDonor(data) {    
+    var groupedData = {};
+    $.each(data, function(i, val) {
+        var name = val.donorName;
+        var amount = val.donationAmount;
+        if (groupedData.hasOwnProperty(name)) {
+            groupedData[name] += amount;
+        } else {
+            groupedData[name] = amount;
+        }
+    });
+
+    var sortedData = [];
+    for (var k in groupedData) sortedData.push([k, groupedData[k]]);
```

Comment:
probably better as an array of objects with ```{name: , value: }``` and then sort that. this took me like 5 mins to figure out what was happening